### ISSUE TITLE
[Feature Request] Related to "int" type in "Advanced Search" - w2grid

### ISSUE THREAD

**Issue by mansurdegirmenci** [vitmalina/w2ui#2053]
I am using asp.net core as "back end". 

In "Advanced Search", it is a problem if special expressions used in int type (for example **=,<,>**) are in the url.
![4](https://user-images.githubusercontent.com/38755096/121789670-6571c600-cbe0-11eb-8c47-ffa29cdf4c32.PNG)

These expressions are used in the url for different purposes. For example, the use of "=" is different. As a result, errors occur.

"=" for example;
![1](https://user-images.githubusercontent.com/38755096/121789732-06608100-cbe1-11eb-863b-99a97417188b.PNG)

">" for example;
![3](https://user-images.githubusercontent.com/38755096/121789742-255f1300-cbe1-11eb-9915-460c9debf2e5.PNG)

Of course this can be solved in "asp.net core". But the use of these expressions did not make sense to me. I leave it to your discretion.

**Comment by mansurdegirmenci** [comment]
alternative nomenclature;

"=" : equal
">" : greaterthan
"<" : lessthan
">=" : greaterthanorequal
"<=" : lessthanorequal

**Comment by mpf82** [comment]
I do understand that certain characters are "bad" in the url, however I do not understand how to reproduce this issue.

Take this demo: http://w2ui.com/web/demos/#/grid/3

Searching for ID=123 results in the following request, with special characters being urlencoded:

    http://w2ui.com/web/demos/data/list.json?request=%7B%22limit%22%3A100%2C%22offset%22%3A0%2C%22searchLogic%22%3A%22AND%22%2C%22search%22%3A%5B%7B%22field%22%3A%22recid%22%2C%22type%22%3A%22int%22%2C%22operator%22%3A%22%3D%22%2C%22value%22%3A123%7D%5D%7D

the decoded request looks like this:

    {"limit":100,"offset":0,"searchLogic":"AND","search":[{"field":"recid","type":"int","operator":"=","value":1}]}

Please provide a minimal fiddle that demonstrates the issue.

**Comment by mansurdegirmenci** [comment]
yes your comment is correct. There is no problem with the url. The problem is on the server side. It treats the equals statement as if the parameter was assigned a value. but we do not value the parameter on the "client side" side.

So what can you ask? Server side is none of my business. Yes true. but the problem is that "between" is a "string" expression along with "=,>,<" these expressions are numeric. that is, the expression "between" needs to be made numerical or it is necessary to change the expressions.

![image](https://user-images.githubusercontent.com/38755096/133830875-59481e47-e3b6-43f8-9ffc-45a2a6c2bd20.png)

I want to point out that you can definitely know. When we examine the big data grid of the library known with extjs, you will see that the naming is like this. It made more sense to me to send it to the server side with these expressions (equal, greater, lessthan) in the url. Thank you for your interest.

**Comment by mansurdegirmenci** [comment]
Example;
"operator" :"equal" or "operator" :"lessthan".
